lib/matplotlib/image.py

```diff
@@ -808,7 +808,7 @@ def get_filterrad(self):
 
 class AxesImage(_ImageBase):
     """
-    An image attached to an Axes.
+    An image, i.e. data on a regular grid, attached to an Axes.
```

Comment:
```suggestion
    An image with pixels on a regular grid, attached to an Axes.
```
This is the same language as you used in `NonUniformImage`, and I think a bit better here?